.github/CODEOWNERS

```
@@ -0,0 +1,18 @@
+# Testing
+*.test.ts @Tlacenka
+/e2e/ @Tlacenka
+/testing/ @Tlacenka
+vite.config*.ts @Tlacenka
+
+# Mocks
+mocks/ @Tlacenka
+/examples/react-todos-app/ @matejchalk
+/testing/test-utils/src/lib/utils/dynamic-mocks/ @matejchalk
+
+# Plugins
+/packages/plugin-eslint/ @matejchalk
+/packages/plugin-coverage/ @Tlacenka
+/packages/plugin-lighthouse/ @BioPhoton
+/examples/plugins/src/file-size/ @BioPhoton
+/examples/plugins/src/lighthouse/ @BioPhoton
+/examples/plugins/src/package-json/ @BioPhoton
```

Maybe just auto-assign Michael to all example plugins by default? We can add specific exceptions later if needed.

```suggestion
/examples/plugins/ @BioPhoton
```